docs/csi-addons-config.md

```diff
@@ -0,0 +1,26 @@
+# CSI-Addons Operator Configuration
+
+CSI-Addons Operator can consume configuration from a ConfigMap named `csi-addons-config`
+in the same namespace as the operator. This enables configuration of the operator to persist across
+upgrades. The ConfigMap can support the following configuration options:
+
+| Option                        | Default value   | Description                                   |
+| ----------------------------- | --------------- | --------------------------------------------- |
+| `reclaim-space-timeout`       | `"3m"`          | Timeout for reclaimspace operation            |
+| `max-concurrent-reconciles`   | `"100"`         | Maximum number of concurrent reconciles       |
+
+## Example
```

Comment:
let's not duplicate the configmap here as it adds a burden to maintain at 2 places, just add a reference to the configmap here.